docs/guides/dpdk-guide.md

```diff
@@ -41,13 +41,9 @@ config.
 
 #### Set environment variables
 
-- export DEPEND_INSTALL=`absolute path for installing dependencies`
+- export P4CP_DEP_INSTALL=`absolute path for installing P4 Control plane dependencies`
```

Comment:
`DEPEND_INSTALL` is like `SDE_INSTALL`. It is used by both CMake and the helper scripts.

The only one of the three environment variables you can rename is `IPDK_INSTALL`. The others have broad significance.

Reply:
Got it, removed the changes since they are not meant to be part of this PR. Accidently pushed them here.